### PR TITLE
Create event card & loading UI components for Map page

### LINKED ISSUE #39: Add loading view in Map page
There is a time gap until we get google map and I think we can add some loading view for it.

https://user-images.githubusercontent.com/67321065/255040032-9fbae674-4a81-47cf-b5f2-c7313818fceb.mov

### LINKED ISSUE #43: Create map modal

### PR DESCRIPTION
**Pull request by devjoylee**
### Feature Description

- Add `events.ts` as dummy data. (this can be deleted later)
@SachiGoto Please help add more data in this file 

- Map Loading UI completed
 The loading page will be shown at least 2 secs or more.

- Event Card UI completed

### Related Issue

closes #39 
closes #43 

### Something you want to mention

Suggestion, can we change  the component name **`Card` > `FavCard`** (in **favlist - components - Card**)
so that we don't get confused with `EventCard` component

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Lada496 — APPROVED** [review]
Thank you @devjoylee I like the loading image!

File: src/app/map/page.tsx [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+import { Loading } from "./components/Loading";
+import { useState } from "react";
 
 const MapPage = () => {
-  return <GoogleMaps />;
+  const [isLoading, setIsLoading] = useState(true);
+  const [mapLoaded, setMapLoaded] = useState(false);
+
+  setInterval(() => {
+    mapLoaded && setIsLoading(false);
```
Comment (keento0809) [inline src/app/map/page.tsx:12]: It's pretty cool to use `setInterval` with the loading status!

File: src/events.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    location: "The Golden Horn Turkish Bakery & Cafe",
+    description:
+      "Join us a regular monthly social for JavaScript enthusiasts. Whether you're a seasoned JavaScript developer or just starting out, Coffee.js is the perfect opportunity to network with like-minded individuals, share your ideas and learn from others.",
+    imageUrl:
+      "https://secure.meetupstatic.com/photos/event/c/7/2/3/clean_511430979.jpeg",
+    website:
+      "https://www.meetup.com/vancouver-javascript-developers/events/292653989/",
+  },
+  // {
+  //   title: "May, 2023: Coffee.js",
```
Comment (keento0809) [inline src/events.ts:14]: @devjoylee You can delete comment outs if you're not going to use them in the future, but if not, just ignore this comment:pray:

**Comment by keento0809** [comment]
@devjoylee Thank you for your contribution to prepare dummy event data and loader component for google map! 👍  I just left one comment so plz take a look when you have time 🙏

**Review by SachiGoto — APPROVED** [review]

**Pull request merged**